engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -1038,14 +1048,14 @@ private void checkContainerStatus()
   private AllocateResponse sendContainerAskToRM(List<ContainerRequest> containerRequests, List<ContainerRequest> removedContainerRequests, List<ContainerId> releasedContainers) throws YarnException, IOException
   {
     if (removedContainerRequests.size() > 0) {
-      LOG.info(" Removing container request: " + removedContainerRequests);
+      LOG.info(" Removing container request: (size={}) {}", removedContainerRequests.size(), removedContainerRequests);
```

Comment:
Why is size necessary? There will be INFO level entry for each element in the removedContainerRequests? Can we remove this logging completely?